src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Models/TestDto.cs

```diff
@@ -39,5 +50,36 @@ public class TestDto
         public string DiscountLogoPath { get; set; }
 
         public bool HasUserAnswered { get; set; }
+
+        private static bool SetIsAdvent(Microsoft.Extensions.Configuration.IConfiguration configuration)
+        {
+            var isAdventEndDate = configuration.GetSection("Competition:EndDate").Value;
+            var isAdventStartDate = configuration.GetSection("Competition:StartDate").Value;
+            ValidateResultPublicationDateTime(isAdventEndDate);
+            ValidateResultPublicationDateTime(isAdventStartDate);
+
+            CultureInfo culture = new CultureInfo("en-Us");
+
+            if (Convert.ToDateTime(isAdventStartDate, culture) >= DateTime.Now && Convert.ToDateTime(isAdventEndDate, culture) <= DateTime.Now)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        private static void ValidateResultPublicationDateTime(string resultPublicationDateTimeValue)
```

Comment:
why do we have this method? Let's focus on `IsAdvent` logc, we handle results later ;)